zktraffic/tests/test_stats_daemon.py

```diff
@@ -28,6 +30,20 @@
 from twitter.common.http.diagnostics import DiagnosticsEndpoints
 
 
+class FakeTimer(Timer):
```

Comment:
not, because that's basically ignored. only tick() and a subsequent call to after() can affect it - makes sense?